rfc/0067-extended-sdk-observability-rto-v2.md

```diff
@@ -952,12 +970,49 @@ If this information is not available for any reason (generally this will only be
+
+| | Experimental (v0) | Stable (v1) | Note |
+| -- | ------------ | ------ | ---- |
+| System name | db.system | db.system.name | |
+| Bucket name | db.name | db.namespace | |
+| Query statement | db.statement | db.query.text | |
+| Operation name | db.operation | db.operation.name | |
+| Error | outcome | error.type | (1) |
+| OSI transport layer | net.transport | network.transport | (2) |
+| Local hostname/IP | net.host.name | Removed | |
+| Local port | net.host.port | Removed | |
+| Server hostname/IP | net.peer.name | server.address | |
+| Server port | net.peer.port | server.port | |
+| Peer address | Did not exist | network.peer.address | |
+| Peer port | Did not exist | network.peer.port | |
+
+**Notes:**
+
+1. In v0, one of the possible values was "Success". In v1, the attribute should be omitted if there was no error.
+2. In v0, the value was "TCP.IP". In v1, it should be "tcp" instead.
+
+## Opting into the stable semantic conventions
+
+Existing implementations should not change the conventions that they emit by default. Users can opt into the new conventions either using a `observability_stable_semantic_convention_opt_in` (`String[]` type) cluster option. Support should also be added for the `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable (values are comma-separated strings) which is  recognized by many OpenTelemetry instrumentation libraries. If both are set, the cluster option has higher precedence.
+
+The values that are recognized from the list provided by `observability_stable_semantic_convention_opt_in`, or `OTEL_SEMCONV_STABILITY_OPT_IN` are:
+* `database`: Emit the stable conventions (the ones outlined elsewhere in this document), and stop emitting the ones emitted by default.
+* `database/dup`: Emit both the stable conventions, in addition to the existing ones.
+
+If both `database` and `database/dup` are present, `database/dup` has higher precedence.
+
+The client should handle switching between the old and the stable conventions internally, and call the relevant methods on the `RequestTracer` and `Meter` interfaces with the right attributes as specified by the option & environment variable above.
+
 # Language Specifics
 
 ## Go
 
-* The Go SDK supports only kv for orphan response logging, all HTTP requests within the Go SDK are performed using the Golang HTTP library, which is synchronous. As such orphaned responses are not possible for non-kv services.
-* The Go SDK architecture is such that on timeouts the requests are removed from internal queueing systems (which is in fact how orphaned responses are detected, the lookup to find the corresponding request fails). This means that only a subset of the orphan logger fields can ever be exposed.
+* The Go SDK supports only KV for orphan response logging, all HTTP requests within the Go SDK are performed using the Golang HTTP library, which is synchronous. As such orphaned responses are not possible for non-kv services.
```

Comment:
Since HTTP requests are cancellable, unlike KV requests, shouldn't all SDKs have this limitation?  The only way I could imagine getting orphan responses from non-KV services would be if SDKs were continueing to process HTTP requests in the background in spite of them timing out.